ci/etc/integration-tests-config.ps1

```diff
@@ -41,6 +41,9 @@ $env:GOOGLE_CLOUD_CPP_SPANNER_TEST_SERVICE_ACCOUNT="spanner-iam-test-sa@${env:GO
 # Cloud Pub/Sub only needs GOOGLE_CLOUD_PROJECT
 $env:GOOGLE_CLOUD_CPP_PUBSUB_TEST_QUICKSTART_TOPIC="quickstart"
 
+# Cloud BigQuery only needs GOOGLE_CLOUD_PROJECT
```

Comment:
I'm not sure what this is supposed to imply (nor the previous Pub/Sub one), as it seems they need at least these extra parameters.  Perhaps they would both be better just as
```
# Cloud <PROJECT> configuration parameters
```
?